test/WebJobs.Script.Tests.Integration/Host/ScriptHostManagerTests.cs

```diff
@@ -141,6 +149,9 @@ public ScriptHostManagerTests()
 
                         // rename directory & delete old blob
                         Directory.Move(oldDirectory, newDirectory);
+
+                        resetEvent.Wait(TimeSpan.FromSeconds(60));
```

Comment:
Ah, yeah. I'll bring that down. I bumped it up to buy me some time while debugging some of this stuff and forgot to set it back.